azurerm/data_source_firewall_test.go

```diff
@@ -21,8 +21,8 @@ func TestAccDataSourceAzureRMFirewall_basic(t *testing.T) {
 			{
 				Config: testAccDataSourceFirewall_basic(rInt, location),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(dataSourceName, "ip_configuration.0.name", "configuration"),
-					resource.TestCheckResourceAttrSet(dataSourceName, "ip_configuration.0.private_ip_address"),
+					resource.TestCheckResourceAttr(dataSourceName, "ip_configurations.0.name", "configuration"),
```

Comment:
Please revert all of these references back to `ip_configuration`

Reply:
reverted.